pandas/tests/reshape/test_pivot.py

```diff
@@ -142,6 +142,25 @@ def test_pivot_table_dropna_categoricals(self, dropna):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_with_non_observable_dropna_civ(self, dropna):
+        # gh-21370
+        arr = [np.nan, 'low', 'high', 'low', 'high', 'high', np.nan]
+        df = pd.DataFrame({"In": pd.Categorical(arr,
```

Comment:
Even if this technically passes LINTing I find the indentation very difficult to read. Can you try putting each key of the dict at the start of the line to see if that improves readability?

Reply:
alright, I will look into it